include/mbgl/util/projection.hpp

```diff
@@ -13,7 +13,7 @@ namespace mbgl {
 class Projection {
 public:
     // Map pixel width at given scale.
-    static double worldSize(double scale) {
+    static constexpr double worldSize(double scale) {
```

Comment:
@jfirebaugh my understanding is that using `constexpr` allows clearer understanding of what we want to do, and what the compiler is possible of calculating in compile time - I personally find class/struct ctors using `constexpr` to be a more elegant pattern too.

There are specific cases where `constexpr` usage is bogus on GCC 4.9 - though this is not the case for simple cases like the ones above i.e. we still use plain `constexpr` in lots of places in our https://github.com/mapbox/mapbox-gl-native/tree/qt-staging branch.

As example, there are cases where `constexpr` works fine in GCC 4.9 - like this [example](https://godbolt.org/g/PQ4fRk), and cases [where it doesn't](https://godbolt.org/g/zmmCg8). 

Probably the best way to make sure we're compliant with GCC 4.9 is to have a GCC 4.9-based build bot running as part of our CI, but I guess that's out of scope for this PR.